frontend/src/modules/entitydetails/components/Metadata.js

```diff
@@ -202,6 +202,14 @@ export default class Metadata extends React.Component<Props> {
         return this.renderSourceObject(entity.source);
     }
 
+    navigateToPath = (event: SyntheticMouseEvent<>) => {
+        event.preventDefault();
+
+        // $FLOW_IGNORE
```

Comment:
Please explain why you ignore Flow here.

Reply:
See above.